src/core/exec-invoke.c

```diff
@@ -4302,13 +4302,18 @@ int exec_invoke(
                 return log_exec_error_errno(context, params, r, "Failed to set up standard input: %m");
         }
 
-        r = setup_output(context, params, STDOUT_FILENO, socket_fd, named_iofds, basename(command->path), uid, gid, &journal_stream_dev, &journal_stream_ino);
+        _cleanup_free_ char *fname = NULL;
+        r = path_extract_filename(command->path, &fname);
+        if (r < 0)
+                return log_warning_errno(r, "Failed to extract filename from path '%s', ignoring: %m", command->path);
```

Comment:
Needs to use log_exec_error_errno like other logs above and below, and also it's not ignored so it shouldn't say "ignoring"